src/main/java/core/basesyntax/service/impl/FruitReportImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitReport;
+import java.util.List;
+
+public class FruitReportImpl implements FruitReport {
+    private final FruitDao fruitDao;
+
+    public FruitReportImpl(FruitDao fruitDao) {
```

Comment:
pass FruitService instead of FruitDao, inside of FruitService should be FruitDao, in FruitService should be method getAll too

Reply:
fixed